packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -157,8 +161,11 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
             return;
         }
 
+        // if the popover handle exists, take it into account
+        const popoverHandleAdjustmentFactor = this.state.isTruncated ? this.popoverComponent.state.targetWidth : 0;
```

Comment:
clever to reuse the already-computed state, but also somewhat sketchy to rely on state field from component instance...